src/App.js

```diff
@@ -1,43 +1,12 @@
-import "./App.css";
-import logo from "./logo.png";
+import './App.css'
 
+import { Home } from './components/home/Home.jsx'
```

Comment:
Could try barrel imports as it is more clean as components increase

Reply:
Sure